src/service/overseas.entities.service.ts

```diff
@@ -7,11 +7,13 @@ import { Transactionkey, OverseasEntityKey } from "../model/data.types.model";
 import { makeApiCallWithRetry } from "./retry.handler.service";
 import { ApplicationData } from "../model/application.model";
 
+// TODO find all calls to this function and remove the isSaveAndResumeFeatureActive
+// parameter when registration and update save and resume flags are removed
 export const createOverseasEntity = async (
   req: Request,
   session: Session,
   transactionId: string,
-  isSaveAndResumeFeatureActive: boolean = false
+  isSaveAndResumeFeatureActive: boolean
```

Comment:
removed the default of false here, and changed the calls that passed no value to pass false. This will be removed when update S&R flag removed.